lang/cs/Org.Apache.REEF.Wake/Remote/Impl/DefaultRemoteManager.cs

```diff
@@ -88,8 +93,9 @@ public DefaultRemoteManager(IPEndPoint localEndpoint, ICodec<T> codec)
         /// <param name="localAddress">The address to listen on</param>
         /// <param name="port">The port to listen on</param>
         /// <param name="codec">The codec used for serializing messages</param>
+        /// <param name="tcpPortProvider">provides port numbers to listen</param>
         [Obsolete("Use IRemoteManagerFactory.GetInstance() instead.", false)]
```

Comment:
The factory is the only place that is allowed to call it. After 0.12 is out, we can make this constructor `internal`.